### ISSUE TITLE
Ubuntu GUI login screen does not appear when nu set as shell with chsh

### ISSUE THREAD

**Issue by rofrol** [nushell/nushell#934]
**Describe the bug**

Cannot login to GUI Ubuntu when `nu` set as default shell with `chsh`.

**To Reproduce**
1. `> config --set [path $nu:path]`
2. `> config --set [env $nu:env]`
3. `sudo chsh -s /home/user/.cargo/bin/nu user`

**Expected behavior**
You see login screen.

**Configuration (please complete the following information):**
 - OS: Ubuntu 19.10
 - Version 0.5.0

**Comment by jonathandturner** [comment]
Did you happen to also add Nu as a shell in `/etc/shells`?

**Comment by DeadMozay** [comment]
openSUSE Tumbleweed GNOME, same problem, in `/etc/shells` added

**Comment by DeadMozay** [comment]
The situation is strange, I checked it on another PC with the same OS, there is no such problem, there are only differences in the set of installed software, I did not find any errors in the logs that would indicate the reason. Does Nushell keep its logs? I would like to deal with the problem, on the problem PC this error was always

**Comment by alexandrestein** [comment]
I'm also affected by this issue on Debian 10.
I don't know if it can be related but I'm running on Wayland.

**Comment by zbraniecki** [comment]
same issue on ubuntu 20.10. Should it be part of #261?

**Comment by DeadMozay** [comment]
Perhaps this error appears if nushell was compiled without `--features=extra`, compiled with `--features=extra`, the system booted without problems

**Comment by jonathandturner** [comment]
@DeadMozay - I wonder which feature fixes the issue with login. Looking through the list, nothing jumps out as commands it might use (maybe `start`?) but maybe some feature pulls in something it needs.

**Comment by DeadMozay** [comment]
To be honest, I myself did not understand why this error occurred, and why it disappeared, there were no errors in any log, at first I thought that the problem was in GDM, after entering the password, the session was thrown back to GDM, I began to check the system logs and GDM itself, for the presence of at least some information, but there is nothing, then I began to check on another PC with the same OS, and there was no such problem, there was an option with some kind of missing package, I began to compare the installed packages on two PCs, in general the result this also did not work, today I decided to check it again on the problem PC, the original version caused the same error, the user session in the GUI did not start, and there were no errors in the tty console, it started without problems, compiled `--features=extra`, and the error disappeared , the system started in the GUI, the only difference is when building with `--features=extra`, an additional dependency `pkgconfig(x11)` was requested for one of the crates, but since rust compiles the dependencies statically, but then the error It should have always appeared.

**Comment by zbraniecki** [comment]
I don't think this is fixed. Did anyone try lately?

**Comment by DeadMozay** [comment]
It seems that the problem is somewhere with permissions, I tried it in KDE, the situation is similar there (nushell 0.34)
```
[   868.473] (EE) xf86OpenConsole: Cannot open virtual console 1 (Permission denied)
[   868.473] (EE) 
[   868.473] (EE) 
Please consult the The X.Org Foundation support 
	 at http://wiki.x.org
 for help. 
[   868.473] (EE) 
[   868.473] (WW) xf86CloseConsole: KDSETMODE failed: Bad file descriptor
[   868.473] (WW) xf86CloseConsole: VT_GETMODE failed: Bad file descriptor
[   868.556] (EE) Server terminated with error (1). Closing log file.
```

**Comment by fdncred** [comment]
is this still an issue with the latest main?

**Comment by DeadMozay** [comment]
> is this still an issue with the latest main?

Yes, x11 is still relevant, with wayland it starts without problems

**Comment by Myrddin-Wyllt** [comment]
- Is this issue exclusive to shells installed in a home directory?
- Is sudo necessary to change shells? The ArchWiki doesn't call for this.

I just tested this today, it works fine here; in my case, I used nushell from the repos (i.e., it was installed to /usr/bin/nu). If I find time to test it in a home directory, I'll follow up.

I don't think the display manager has access to a user's home directory.

**Comment by uncomfyhalomacro** [comment]
Is this issue still open?
